app/src/server/api/routers/datasetEntries.router.ts

```diff
@@ -780,6 +778,15 @@ export const datasetEntriesRouter = createTRPCRouter({
                 .select((eb) => [
                   "dede.datasetEntryId as datasetEntryId",
                   eb.fn.agg<number>("AVG", [`der.score`]).as(`scoreForEval_${datasetEval.id}`),
+                  sql
+                    .raw("CAST(COUNT(CASE WHEN der.score = 1 THEN 1 ELSE NULL END) AS INT)")
+                    .as(`wins_${datasetEval.id}`),
+                  sql
+                    .raw("COUNT(CASE WHEN der.score = .5 THEN 1 ELSE NULL END)")
+                    .as(`ties_${datasetEval.id}`),
+                  sql
+                    .raw("COUNT(CASE WHEN der.score = 0 THEN 1 ELSE NULL END)")
+                    .as(`losses_${datasetEval.id}`),
```

Comment:
You're more familiar with kysely sql syntax than I am. I think the updated code should fix the vulnerability that you identified, but I would appreciate you double-checking.

Reply:
I'm still seeing a lot of sql.raw statements with templated strings -- those are the things to avoid. I think the way you're using those can hopefully all be replaced by sql`` tags.